client/jetpack-cloud/sections/partner-portal/license-preview/index.tsx

```diff
@@ -101,12 +101,18 @@ export default function LicensePreview( {
 							</span>
 						) }
 
-						{ justIssued && (
+						{ justIssued && ! justAssigned && (
 							<span className="license-preview__tag license-preview__tag--is-just-issued">
 								<Gridicon icon="checkmark-circle" size={ 18 } />
 								{ translate( 'Just issued' ) }
 							</span>
 						) }
+						{ justAssigned && (
+							<span className="license-preview__tag license-preview__tag--is-assigned">
+								<Gridicon icon="checkmark-circle" size={ 18 } />
+								{ translate( 'Successfully assigned' ) }
+							</span>
```

Comment:
This is currently not showing up I'm assuming it will after the `highlight` query param feedback has been looked at 😄